### PR TITLE
Add QMEU to goreleaser

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

**Comment by awgreene** [comment]
Tested the goreleaser github action here: https://github.com/awgreene/operator-lifecycle-manager/actions/runs/4410163118/jobs/7727334061

**Comment by awgreene** [comment]
Left the experiment env var in, should have removed it.
/hold

**Comment by awgreene** [comment]
Works fine without the experimental env var.
/unhold

File: .github/workflows/goreleaser.yaml [hunk 1]
```diff
@@ -37,6 +37,9 @@ jobs:
           tag_name: ${{ github.ref }}
           release_name: ${{ github.ref }}
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
```
Comment (tmshort) [inline .github/workflows/goreleaser.yaml:42]: So, this is needed due to support different platforms, or??

Comment (tmshort) [reply .github/workflows/goreleaser.yaml:42]: (Or to make docker happy?)

File: .github/workflows/goreleaser.yaml [hunk 2]
```diff
@@ -37,6 +37,9 @@ jobs:
           tag_name: ${{ github.ref }}
           release_name: ${{ github.ref }}
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```
[the remark this answers is not in the document]
Comment (awgreene) [reply .github/workflows/goreleaser.yaml:41]: That's a great idea @m1kola, will update shortly.

[the remark this answers is not in the document]
Comment (m1kola) [reply .github/workflows/goreleaser.yaml:41]: Probably out of scope for this PR - do we need this workflow to run on PRs at all? It looks like most of the steps are being skipped and only checkout, go setup and QEMU setup are being performed

image

[the remark this answers is not in the document]
Comment (awgreene) [reply .github/workflows/goreleaser.yaml:41]: Done!

[the remark this answers is not in the document]
Comment (tmshort) [reply .github/workflows/goreleaser.yaml:41]: That likely only applies when we need to use/get the image tag. (e.g. `Get the image tag` above). In this case, there's no different course of action, we always want to install QEMU, as it only has local significance.

[the remark this answers is not in the document]
Comment (tmshort) [reply .github/workflows/goreleaser.yaml:41]: I would also say this doesn't need to be run on PRs.

**Comment by m1kola** [comment]
/lgtm

Probably out of scope for this PR - do we need this workflow to run on PRs at all? It looks like most of the steps are being skipped and only checkout, go setup and QEMU setup are being performed

image

**Comment by tmshort** [comment]
I agree with @m1kola that this doesn't need to be run on PR.
Relatively simple (future) change:
https://github.com/operator-framework/operator-lifecycle-manager/blob/56c2a5f0ec33f3c1792c5bc1f482a3cc50502eee/.github/workflows/goreleaser.yaml#L3
/lgtm
/appove

**Review by tmshort — APPROVED** [review]

**Comment by tmshort** [comment]
I'm assuming you did a rebase.
/lgtm

**Pull request merged**